components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -29,7 +30,11 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
   pageSize = 10;
   totalReports = 0;
   firstReportIsLoaded = false;
-
+  downloadList: Array<string> = [];
+  downloadOptsVisible: boolean = false;
+  downloadStatusVisible: boolean;
+  downloadInProgress: boolean;
+  downloadFailed: boolean;
```

Comment:
These should all be consistently initialized; also with an initialization value the type is redundant.
```suggestion
  downloadOptsVisible = false;
  downloadStatusVisible = false;
  downloadInProgress = false;
  downloadFailed = false;
```